src/components/BlogList/BlogList.jsx

```diff
@@ -40,13 +51,22 @@ const BlogList = ({ blogs, query }) => {
         ))}
       </div>
       <div className="blog-list">
-        {filteredBlogsByTags.length
-          ? filteredBlogsByTags?.map((blog) => (
-              <BlogCard key={blog.id} blog={blog} />
-            ))
+        {currentBlogs?.length
+          ? currentBlogs?.map((blog) => <BlogCard key={blog.id} blog={blog} />)
           : "Blog not found"}
       </div>
-    </>
+      {blogsPerPage < totalBlogs && (
+        <div className="button-wrapper">
+          <Button
+            type={"button"}
+            onClickHandler={handleLoadMore}
+            className="load-more-button"
+          >
+            Load More
```

Comment:
I don't see any `Show less` functionality here. please add that too.

Reply:
added